tensorflow/compiler/mlir/lite/ir/tfl_traits.h

```diff
@@ -22,6 +22,11 @@ limitations under the License.
 #include "mlir/Support/LLVM.h"  // TF:local_config_mlir
 #include "tensorflow/compiler/mlir/lite/utils/quantization_utils.h"
 
+// exp10() is not a standard function
```

Comment:
Could you update the call site to use pow instead?

Reply:
sure.